lib/common/issues/src/dashboard.rs

```diff
@@ -0,0 +1,175 @@
+use std::collections::HashSet;
+use std::sync::{Mutex, MutexGuard, OnceLock};
+
+use serde::{Serialize, Serializer};
+
+use crate::issue::{DummyIssue, Issue};
+
+#[derive(Default, Serialize)]
+struct Dashboard {
+    pub issues: HashSet<Box<dyn Issue>>,
```

Comment:
It seems the Hash & Eq is determined solely by the `code` field.
We could use a `Hashmap<code, Issue>` instead to keep a generic definition of hash and equality on the `Issue` type.

Reply:
Great point 😄

Reply:
agreed, making custom Hashing looks hacky

Reply:
Turned into HashMap. It does make it clearer